js/index.js

```diff
@@ -0,0 +1,38 @@
+function getRandomInt(min, max) {
+	min = Math.ceil(min);
+	max = Math.floor(max);
+	return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+let dice = {
+	value: 1,
+	roll: () => {
+		this.value = getRandomInt(1, 6);
+		return this.value;
+
+	}
+};
+
+let dice2 = {
+	value: 1,
+	roll: () => {
+		this.value = getRandomInt(1, 6);
+		return this.value;
+
+	}
+};
+function printimage(image, image1){
+	let containar = document.querySelector("#dice1");
+	let containar1 = document.querySelector("#dice2");
+
+	containar.style["background-image"]=`url(/images/inverted-dice-${image}.svg)`;
+	containar1.style["background-image"]=`url(/images/inverted-dice-${image1}.svg)`;
+}
+
+let button = document.querySelector("#rolling-button");
+button.addEventListener("click" , () => {
+	let result = dice.roll();
```

Comment:
You don't actually have to store the return of the dice into a variable. You can also just roll the dice and then pass `dice.value` into the `printimage()` function below, since the role method **actually** changes the `value` property on the dice object. Like this:

```javascript
dice.roll();
dice2.roll();

printimage(dice.value, dice2.value);